src/map/pc.cpp

```diff
@@ -13055,7 +12995,7 @@ TIMER_FUNC(pc_bonus_script_timer){
 * @param flag: Reason to remove the bonus_script. e_bonus_script_flags or e_bonus_script_types
 * @author [Cydh]
 **/
-void pc_bonus_script_clear(struct map_session_data *sd, uint16 flag) {
+void pc_bonus_script_clear(struct map_session_data *sd, enum e_bonus_script_flags flag) {
```

Comment:
This is not a fixed enumerated value but a bitmask and should be uint32 instead.